src/schema/geneInteraction.yaml

```diff
@@ -0,0 +1,467 @@
+      subject:
+        range: Gene                   # is this declaration needed since it is already made in the parent class?
+        values_from:
+          - alliance
+      predicate:
+        subproperty_of: interacts_with
+        symmetric: true
+      relation:
+        values_from:
+          - RO
+          - mi
+      object:
+        range: Gene                   # is this declaration needed since it is already made in the parent class?
+        values_from:
+          - alliance
+      data_provider:
+        range: uriorcurie
+        description: >-
+          The database that curated the interaction. e.g. BioGRID
+        values_from:
+          - mi
+#        exact_mappings: # need to determine how to specify that the values should be an ontology class or descendant
+#          - mi:0461
+        required: true
+        multivalued: false
+      association_publications:
+        description: >-
+          The reference from which the interaction was annotated.
+        multivalued: false
+    exact_mappings:
+      - biolink:pairwise_gene_to_gene_interaction
+
+  GeneMolecularInteraction:
+    description: >-
+      A physical molecular interaction between gene products (e.g. protein-protein
+      interactions or protein-RNA interactions) or between genes and DNA-binding
+      factors (e.g. protein-DNA interactions)
+    is_a: GeneInteraction
+    slots:
+      aggregation_database
+      detection_method
+    slot_usage:
+      aggregation_database:
+        values_from:
+          - mi
+#        exact_mappings: # need to determine how to specify that the values should be an ontology class or descendant
+#          - mi:0461
+        description: >-
+          The database that collected the interaction annotation from another resource and
+          aggregated it into a consolidated resource. e.g. IMEx
+      detection_method:
+        values_from:
+          - mi
+#        exact_mappings: # need to determine how to specify that the values should be an ontology class or descendant
+#          - mi:0001
+      predicate:
+        subproperty_of: molecularly_interacts_with
+        symmetric: true
+      relation:
+        values_from:
+          - RO
+          - mi
+    exact_mappings:
+      - biolink:pairwise_molecular_interaction
+
+  GeneGeneticInteraction:
+    description: >-
+      A genetic interaction between genes (e.g. phenotypic suppression)
+    is_a: GeneInteraction
+    slots:
+      interactor_A_genetic_perturbation
+      interactor_B_genetic_perturbation
+      phenotype_or_trait
+    slot_usage:
+      predicate:
+        subproperty_of: genetically_interacts_with
+
+
+## ------------
+## SLOTS
+## ------------
+
+slots:
+
+  aggregation_database:         # this should be generalized in core.yaml
+    required: false
+    multivalued: false
+    range: uriorcurie
+
+  association_publications:                     # this should be generalized in core.yaml
```

Comment:
can this just be publications?

Reply:
I modeled it after the 'publications' slot in biolink which is written as:

`  publications:
    singular_name: publication
    description: >-
      connects an association to publications supporting the association
    is_a: association slot
    multivalued: true
    range: publication`

This is defined as an association_slot to support an association; Since I didn't see what 'publications' slot existed for nodes, which I presume we need, I thought I would differentiate this slot as "association publications" with the idea that there would be a separate slot "node publications" to handle publications directly associated to a node. I'm fine leaving it as 'publications' but then I think it shouldn't be constrained to only be an association slot for association types.